lib/custom_widgets/row_genre_item.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+import '../text_styles/custom_text_style.dart';
+import '../ui_constants.dart';
+
+class RowGenreItem extends StatelessWidget {
+  const RowGenreItem({super.key, required this.genre});
+
+  final String genre;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: const BoxDecoration(
+        color: Colors.black12,
+        borderRadius: BorderRadius.all(
+          Radius.circular(MovieDetailsUiConstants.borderRadius),
+        ),
+      ),
+      child: Padding(
```

Comment:
The `Container` already has a property named `padding`, so that you can use that one instead of having a dedicated `Padding` widget!

Reply:
done